apps/server/src/modules/lesson/controller/dto/lesson-resources.model.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class LessonResources {
```

Comment:
The naming here is not correct. You should rename the class to 'LernstoreResourcesDto'. And you should rename the file accordingly to 'lernstore-resources.dto.ts'.

Reply:
Done

Reply:
It's gone. This is old version of the file